src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -605,4 +674,18 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
     openLegend() {
         this.showLegend = true;
     }
+
+	handleCBoxValueChange(e) {
+		this.noHexFrame = !this.noHexFrame;
```

Comment:
Wouldn't it be nicer to bind the value of the checkbox to a property? Then we wouldn't have to rely on the switching to work (it seems there could go wrong a lot of stuff). If the binding is, for some reason, not possible, maybe the value is contained in the event `e`? Or accessible via the native element.

Reply:
Yes, it was not possible to do binding even after creating a `formfield-checkbox`. I updated the impl. to use event.